### PR TITLE
109 create webapp creation area page

### PR DESCRIPTION
**Pull request by mathieu-MZ**
## Changes made

Add the my area page and the creation of ne area

## Added/updated tests?

- [x] No,

### CODE REVIEW

File: frontend/src/views/MyAreaPage.vue [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    }
+}
+
+function handleScrollAttemptSecondPage(event: WheelEvent) {
+    if (event.deltaY < 0 && scrollY.value === 0) {
+        isHeroVisible.value = true;
+    }
+}
+
+const actions = ref([
```
Comment (Tugduoff) [inline frontend/src/views/MyAreaPage.vue:191]: It's areas and should be retrieved from the server (ask Adam about it)

On a route entitled : /user/areas/

File: frontend/src/components/AREACreationComponent.vue [hunk 2]
```diff
@@ -0,0 +1,153 @@
+<template>
+    <div class="fixed top-1/2 left-1/2 transform -translate-x-1/2 -translate-y-1/2 z-50 flex flex-col gap-4 p-6 bg-gradient-to-r from-indigo-900 to-blue-500 rounded-lg shadow-lg text-white w-160 max-w-full">
```
Comment (Tugduoff) [inline frontend/src/components/AREACreationComponent.vue:2]: Remove the gradient pls. It does not go at all with the style of the website

File: frontend/src/components/AREACreationComponent.vue [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+    </div>
+  </template>
+
+<script setup lang="ts">
+import { ref, onMounted, defineEmits } from 'vue';
+import { useServiceStore } from '@/stores/service';
+import { Icon } from '@iconify/vue';
+
+const serviceStore = useServiceStore();
+const services = ref<{ name: string; color: string; icon: string; reviews: { rate: number; count: number; }; saves: number; actions: { name: string; description: string; }[]; reactions: { name: string; description: string; }[]; }[]>([]);
```
Comment (Tugduoff) [inline frontend/src/components/AREACreationComponent.vue:104]: Use Service type from @/types/service.ts

File: frontend/src/views/MyAreaPage.vue (hunk 1 above)
Comment (Tugduoff) [reply frontend/src/views/MyAreaPage.vue:191]: You wont have the color or the icon doing that, you must look inside the service store for those info.

**Review by Tugduoff — CHANGES_REQUESTED** [review]

File: frontend/src/views/MyAreaPage.vue (hunk 1 above)
Comment (mathieu-MZ) [reply frontend/src/views/MyAreaPage.vue:191]: we wait adam

**Review by AdamLesage — APPROVED** [review]

**Pull request merged**